.github/CONTRIBUTING.md

```diff
@@ -25,7 +27,8 @@ One important thing distinction is that we are building the Polkadot Runtime Env
 **4. Link your local clone to the fork on your Github repo.**
 
 ```
-$ git remote add your-gossamer-repo https://github.com/<your_github_user_name>/gossamer.git
+$ git init
```

Comment:
hmm, `git init` is needed only if you `go get` it seems, but the previous steps say to clone the repo, in which case the `.git` already is initialized.. could you either remove this or update step 3 to:
```
go get -u github.com/ChainSafe/gossamer
cd $GOPATH/src/github.com/ChainSafe/gossamer
git init
```